static/docs/tutorials/index.md

```diff
@@ -0,0 +1,74 @@
+# Tutorials
+
+## Basic Concepts
+
+Learn basic concepts and features of DVC with interactive lessons:
+
+1. [Data Management](https://katacoda.com/dvc/courses/basics/data) <br/> The
+   core function of DVC is data tracking and management. Let's see how to do it.
+
+2. [Getting the Best Performance](https://katacoda.com/dvc/courses/basics/performance)
+   <br/> It is important to optimize the DVC setup for having the best
+   performance with handling big data files.
+
+3. [Tracking Data Versions](https://katacoda.com/dvc/courses/basics/versioning)
+   <br/> DVC takes advantage of GIT's versioning features to keep track of the
+   data versions.
+
+4. [Sharing Data](https://katacoda.com/dvc/courses/basics/sharing) <br/> DVC
+   facilitates sharing of data between different people that work on the same
+   project.
+
+5. [Stages And Pipelines](https://katacoda.com/dvc/courses/basics/pipelines)
+   <br/> DVC has a built-in way to connect ML steps into a DAG and run the full
+   pipeline end-to-end.
+
+## Interactive Tutorials
+
+Learn how DVC can be used in simple ML scenarios:
+
+1. [Data Versioning](https://katacoda.com/dvc/courses/tutorials/versioning)
+   <br/> Using DVC commands to work with multiple versions of datasets and ML
+   models.
+
+2. [Stages and Pipelines](https://katacoda.com/dvc/courses/tutorials/pipelines)
+   <br/> Using DVC commands to build a simple ML pipeline.
+
+3. [dvc fetch](https://katacoda.com/dvc/courses/examples) <br/> We will use an
+   example project with some data, code, ML models, pipeline stages, as well as
+   a few Git tags. Then we will see what happens with dvc fetch as we switch
+   from tag to tag.
+
+## Native Tutorials
```

Comment:
let's remove the name and move this list to the very top (under `#Tutorials` and before `## Interactive`). This list is small (it won't hide other tutorials, but it's hidden effectively now), it does not require a special name (since we stay within the same domain)